internal/test/flags.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+// This file allows us to share a variable between the test framework and our
```

Comment:
Is there no other way around this? Just curious.

Reply:
I could add the variable to `pkg/test`, but I didn't want to expose that as part of the public API, so I decided it would be better to keep that in a separate internal package. If we feel that it's fine having that variable exposed, we could put it in `pkg/test` instead.

Reply:
That's an okay tradeoff, thanks for the explanation!